lib/pure/terminal.nim

```diff
@@ -29,9 +29,7 @@ when not hasThreadSupport:
   var
     colorsFGCache = initTable[Color, string]()
     colorsBGCache = initTable[Color, string]()
-  when not defined(windows):
```

Comment:
There was a reason this was disabled on Windows.

Reply:
Well, the reason is not valid anymore. ANSI codes are now supported by some Windows terminals and I've exported the procs for generating them for Windows targets too.

Reply:
It's disabled because the styleCache table for Windows is not used and not needed.